src/components/resources/CreateResource.js

```diff
@@ -103,7 +103,7 @@ const CreateResource = () => {
     }
 
     const [mediaTag, setMediaTag] = React.useState('');
-    const mediaTagList = ["", "Youtube", "Stack Overflow", "School Website", "Personal Website", "Spreadsheet"];
+    const mediaTagList = ["Interview Tips", "Youtube", "Stack Overflow", "School Website", "Personal Website", "Spreadsheet"];
```

Comment:
I would add an option where a user can pick no tag. Right now the dropdown is preset to "Interview Tips"

Reply:
Fixed!